consensus/hotstuff/signature/weighted_signature_aggregator.go

```diff
@@ -86,7 +87,7 @@ func (w *WeightedSignatureAggregator) Verify(signerID flow.Identifier, sig crypt
 		return fmt.Errorf("couldn't verify signature from %s: %w", signerID, err)
 	}
 	if !ok {
-		return signature.ErrInvalidFormat
+		return fmt.Errorf("invalid aggregated sig from signer (%s): %w", signerID, signature.ErrInvalidFormat)
```

Comment:
include the signer

Reply:
https://github.com/onflow/flow-go/pull/1347/commits/ceea51c7fa9603a421327f92aaa95841f1a36a87

Reply:
I think this doesn't matter, we wrapped the error, the caller should always call `errors.Is` to check the error, rather than `==` to check the error. 

And it's useful to include context in the error

Reply:
I've added `NewErrInvalidFormat(..)` if that's okay instead of merging this PR if that's okay